FND/Places/Addresses.rdf

```diff
@@ -133,5 +599,120 @@
+		<rdfs:label>has postal code</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>the postal code of this address as specified by the local postal service</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+	</owl:DatatypeProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasPostdirectionalSymbol">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+		<rdfs:label>has postdirectional symbol</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;PostdirectionalSymbol"/>
+		<skos:definition>specifies a geographic directional symbol that follows the street name and street suffix in a street address</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasPredirectionalSymbol">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+		<rdfs:label>has predirectional symbol</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;PredirectionalSymbol"/>
+		<skos:definition>specifies a geographic directional symbol that occurs after the primary street number but before the street name in a street address</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasPrimaryAddressNumber">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+		<rdfs:label>has primary address number</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;PrimaryAddressNumber"/>
+		<skos:definition>specifies a a location with respect to a given street</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasSecondaryUnit">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+		<rdfs:label>has secondary unit</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;SecondaryUnit"/>
+		<skos:definition>specifies an individual unit within a larger structure, such as an apartment, office, hangar, slip, mailbox, and so forth, at a given street address</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasStreetName">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+		<rdfs:label>has street name</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;StreetName"/>
+		<skos:definition>specifies an identifier for a street in some context (e.g., &apos;Baker&apos;, &apos;First&apos;, &apos;Main&apos;)</skos:definition>
```

Comment:
Need to be more clear that, for example, the hasStreetName for Wall Street would be simply "Wall".
This makes me somewhat uncomfortable as being too textual. And would be awkward to allow the street to be represented as a resource in own right - had you considered that? We have Municipality and Building as classes, why not Street? What about areas which are not counties such as Tribeca?

Reply:
I feel safer following the post office standard on this one.  The name and suffix could be combined in an additional element though, if you want me to create a separate class for that.  

For things like 'Tribeca', there is, in the extension for the US that I need to add, the concept of 'Urbanization', which deals with neighborhoods in a city.  That's a separate line item in the address though, and I think is specific to Puerto Rico.  San Francisco has many well known neighborhoods as well - we could add that Neighborhood to the countries class, and then Urbanization would be a subclass of that in the US-specific post office standard.